haiku-games/bong/bong-1.4~git.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="BeOS version of Pong"
+DESCRIPTION="Bong is a FreeWare game for BeOS,based on the old arcade classic Pong"
+HOMEPAGE="https://github.com/HaikuArchives/Bong"
+COPYRIGHT="1997 by Gertjan van Ratingen"
+LICENSE="bong"
+REVISION="1"
+srcGitRev=906d822b79bdae9328fe743dffe2871f96a30e0e
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+SOURCE_FILENAME="bong-1.4~$srcGitRev.tar.gz"
+SOURCE_DIR="Bong-$srcGitRev"
+
+PATCHES="bong-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	bong = $portVersion
+	app:bong = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:gcc
+	cmd:make
+	"
+
+GLOBAL_WRITABLE_FILES="
+	settings/bong auto-merge
+	"
+
+USER_SETTINGS_FILES="
+	settings/bong template data/bong/defaultconfig
```

Comment:
May i ask why did you add this line ? I think leaving those comments there won't hurt at all

Reply:
Sure!
For some unknown reason the compilation on x86_gcc2 is failing with an error inside that `#if 0` ... `#endif`. The dirty workaround I found was to replace all this block by empty lines.

Hmm, maybe there are UTF-8 chars in there. (I haven't checked)
gcc2 does not like unicode...

Reply:
I've just checked the [bong.cpp](https://github.com/HaikuArchives/Bong/blob/master/bong.cpp#L35) file with:
```sh
cat -v bong.cpp  | diff bong.cpp -
```
and this yelds nothing. So this file does not have any non-printable chars.
This means it does not have any unicde character.
So I guess the parse problem is due to gcc2.

I'm writing this because I saw that you inserted the hack to get rid of the bloc, and that's OK for me, since there does not seem to be a better method.
Well, the bloc could be converted to real comments, instead of _disabled code_.

Hmm, it might be worth investigating further to find out on which exact lines gcc2 is complaining.
Maybe it's just on the repeated \#.

Reply:
I don't have any x86_gcc2 machine atm so i can't test it. But if you're OK with it then just leave it like that for now. (Although creating a big hole in the file isn't aesthetically good).
So, can i get approval on gci ?